include/mockgyver.hrl

```diff
@@ -8,19 +8,31 @@
 %% run tests with a mock
```

Comment:
Didn't find how to add this on line 4: dependency no longer needed (unless we refactor the fixture into eunit_addons)?

Reply:
Good point. Let's leave it in.